src/utils/testing.rs

```diff
@@ -15,13 +18,33 @@ use std::io::Read;
 use tracing::error;
 // Utility for unit tests
 
+pub const TEST_CONFIG_FILE: &str = "tests/config.test.yaml";
+
+pub async fn ztf_alert_worker() -> ZtfAlertWorker {
+    // initialize the ZTF indexes
+    let db = conf::build_db(&conf::load_config(TEST_CONFIG_FILE).unwrap())
+        .await
+        .unwrap();
+    initialize_survey_indexes("ZTF", &db).await.unwrap();
+    ZtfAlertWorker::new(TEST_CONFIG_FILE).await.unwrap()
+}
+
+pub async fn lsst_alert_worker() -> LsstAlertWorker {
+    // initialize the ZTF indexes
+    let db = conf::build_db(&conf::load_config(TEST_CONFIG_FILE).unwrap())
+        .await
+        .unwrap();
+    initialize_survey_indexes("LSST", &db).await.unwrap();
+    LsstAlertWorker::new(TEST_CONFIG_FILE).await.unwrap()
+}
```

Comment:
As you mentioned before, mongodb doesn't mind if we try to create the same index more than once, so calling these functions from multiple test threads won't be a problem. At some point we might consider making sure the indexes are only created once, but until then I'm good with this approach.